osu.Game.Rulesets.Taiko/Difficulty/TaikoDifficultyAttributes.cs

```diff
@@ -36,12 +40,21 @@ public class TaikoDifficultyAttributes : DifficultyAttributes
         [JsonProperty("mono_stamina_factor")]
         public double MonoStaminaFactor { get; set; }
 
+        [JsonProperty("rhythm_peak_strains")]
         public double RhythmTopStrains { get; set; }
 
+        [JsonProperty("colour_peak_strains")]
         public double ColourTopStrains { get; set; }
 
+        [JsonProperty("stamina_peak_strains")]
         public double StaminaTopStrains { get; set; }
 
+        /// <summary>
+        /// The factor corresponding to the consistency of a map.
+        /// </summary>
+        [JsonProperty("consistency_factor")]
+        public double ConsistencyFactor { get; set; }
```

Comment:
This is new, but the `JsonProperty` remark is still relevant. If you definitely plan on using this attribute in PP prior to the next deploy, we should database this now so that I don't forget.

Can you confirm that's the intent?

Reply:
Correct, that's the intent, would it be preferable to database these, as well as the above planned?

Reply:
Yes, please database anything you intend on using and then remove `JsonProperty` from any that you aren't.

Reply:
Has been resolved.